pymongoexplain/commands.py

```diff
@@ -63,10 +92,11 @@ def command_name(self):
 class DistinctCommand(BaseCommand):
     def __init__(self, collection: Collection, key, filter, session,
                  kwargs):
-        super().__init__(collection.name)
-        self.command_document = {"key": key, "query": filter}
-        for key, value in kwargs.items():
-            self.command_document[key] = value
+        super().__init__(collection.name, kwargs.get("collation", None))
+        self.command_document.update({"key": key, "query": filter})
+        if kwargs.get("read_concern", None) is not None:
+            self.command_document["read_concern"] = kwargs["read_concern"]
```

Comment:
`read_concern` is not an option for distinct so we can remove this.

Reply:
Done.